.github/workflows/gh-release.yml

```diff
@@ -38,26 +38,44 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: get AvN Parachain collator Artifact
+      - name: get AvN Parachain collator Artifact (Ubuntu 20.04)
         uses: actions/download-artifact@v4
         with:
           name: ${{ inputs.parachain-collator-artifact }}
           path: target/release/
 
-      - name: get AvN Parachain runtime Artifact
+      - name: get AvN Parachain runtime Artifact (Ubuntu 20.04)
         uses: actions/download-artifact@v4
         with:
           name: ${{ inputs.parachain-runtime-artifact }}
           path: target/release/wbuild/avn-parachain-runtime/
 
-      - name: get AvN Parachain test runtime Artifact
+      - name: get AvN Parachain test runtime Artifact (Ubuntu 20.04)
         uses: actions/download-artifact@v4
         with:
           name: ${{ inputs.parachain-test-runtime-artifact }}
           path: target/release/wbuild/avn-parachain-test-runtime/
 
-      - name: add sha256sum to files
-        id: sha256
+      - name: get AvN Parachain collator Artifact (Ubuntu 24.04)
+        uses: actions/download-artifact@v4
+        with:
+          name: parachain-collator
+          path: ubuntu-2404target/release/
+
+      - name: get AvN Parachain runtime Artifact (Ubuntu 24.04)
+        uses: actions/download-artifact@v4
+        with:
+          name: parachain-runtime
+          path: ubuntu-2404/target/release/wbuild/avn-parachain-runtime/
+
+      - name: get AvN Parachain test runtime Artifact (Ubuntu 24.04)
+        uses: actions/download-artifact@v4
+        with:
+          name: parachain-test-runtime
+          path: ubuntu-2404/target/release/wbuild/avn-parachain-test-runtime/
```

Comment:
The release should provide only one version of the wasm, that of 24.04 and their sha256 
Since it's build in web-assembly the outcome can be used across platforms and nodes.
Multiple builds though mess up with the vesrsioning as the build output is not deterministic.